01-getting-started/getting-started.Rmd

```diff
@@ -5,4 +5,107 @@ output:
   html_notebook: 
     toc: true
     toc_float: true
----
\ No newline at end of file
+---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [About refine.bio](#about-refinebio)
+- [About how this tutorial book is structured](#about-how-this-tutorial-book-is-structured)
+- [What you need to install to run the examples](#what-you-need-to-install-to-run-the-examples)
+- [How to get the data](#how-to-get-the-data)
+- [How to use R Markdowns](#how-to-use-r-markdowns)
+- [An important note about file paths and Rmds](#an-important-note-about-file-paths-and-rmds)
+    - [Resources for learning R](#resources-for-learning-r)
+    - [More resources from the CCDL](#more-resources-from-the-ccdl)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## About refine.bio 
```

Comment:
Yes. I'll put a TODO there for now.

Reply:
> These might need to be online links though instead of relative links?

What information do you need to answer this question? To get further along with Jekyll?

Reply:
Probably some Jekyll set up things, but also some insight/guesses on how our users are going to be looking at this? Probably almost always online or downloaded the whole repo?

Reply:
Online links seem safer for now. I think we will need to figure out how easy or hard it is to have individual modules on S3 (#100).